src/app/actions/page.tsx

```diff
@@ -16,11 +17,13 @@ export default async function ActionsPage() {
   return (
     <PageWrapper>
       <ActionsProvider>
-        <ActionsHeader />
-        <ActionsTable />
-        <ActionCreateDialog />
-        <ActionUpdateDialog />
-        <ActionDeleteDialog />
+        <PlacesProvider>
```

Comment:
Do we need the whole context, maybe we could just use `useGetPlaces`? Also, this will load all places on actions page, maybe we could move it to create/update?